src/tactic/positivity.lean

```diff
@@ -297,18 +297,41 @@ private lemma div_nonneg_of_nonneg_of_pos [linear_ordered_field R] {a b : R} (ha
   0 ≤ a / b :=
 div_nonneg ha hb.le
 
+private lemma int_div_self_pos {a : ℤ} (ha : 0 < a) : 0 < a / a :=
+by { rw int.div_self ha.ne', exact zero_lt_one }
+
+private lemma int_div_nonneg_of_pos_of_nonneg {a b : ℤ} (ha : 0 < a) (hb : 0 ≤ b) : 0 ≤ a / b :=
+int.div_nonneg ha.le hb
+
+private lemma int_div_nonneg_of_nonneg_of_pos {a b : ℤ} (ha : 0 ≤ a) (hb : 0 < b) : 0 ≤ a / b :=
+int.div_nonneg ha hb.le
+
+private lemma int_div_nonneg_of_pos_of_pos {a b : ℤ} (ha : 0 < a) (hb : 0 < b) : 0 ≤ a / b :=
+int.div_nonneg ha.le hb.le
+
 /-- Extension for the `positivity` tactic: division is nonnegative if both numerator and denominator
 are nonnegative, and strictly positive if both numerator and denominator are. -/
 @[positivity]
 meta def positivity_div : expr → tactic strictness
-| `(%%a / %%b) := do -- TODO handle eg `int.div_nonneg`
+| `(%%a / %%b) := do
```

Comment:
An alternative approach to the current 8-line match statement would be to handle the `ℤ` case separately, which could look like:
```suggestion
| `(@has_div.div int _ %%a %%b) := do ...
| `(%%a / %%b) := do ...
```
I think there's not really enough overlap between the two cases to warrant keeping them next to each other.